### PR TITLE
additional updates to the SGA data model

### PR DESCRIPTION
**Pull request by moustakas**
I'll need this branch up until I've finished the SGA data release, but this PR should not hold up #136.

### CODE REVIEW

**Comment by moustakas** [comment]
@geordie666 @weaverba137 this PR is ready for your review.

File: pages/sga/sga2020.rst [hunk 1]
```diff
@@ -416,21 +416,25 @@ Specifically, we carry out the following steps for each galaxy group:
...
    semi-major axis in arcsec. In our analysis we take the radius scale factor
-   :math:`r_{0}=10` arcsec to be fixed.
+   :math:`r_{0}=10` arcsec to be fixed. Note that in the limit
+   :math:`r\rightarrow\infty`, :math:`m_{tot}` is the total, integrated
+   magnitude.
 
-   Note that in the limit :math:`r\rightarrow\infty`, :math:`m_{1}` is the
-   total, integrated magnitude, and as :math:`r\rightarrow0`,
-   :math:`m_{0}+m_{1}` is the brightness at the center of the galaxy.
+   With this model, the half-light radius, :math:`r_{50}` can be inferred from
```
Comment (yymao) [inline pages/sga/sga2020.rst:432]:
```suggestion
   With this model, the half-light semi-major axis, :math:`r_{50}` can be inferred from
```

**Review by geordie666 — APPROVED** [review]
This looks good to me from a coding perspective @moustakas. While I was reviewing the `sga2020` page I found a handful of spelling/grammar/linking/rendering/consistency issues that you might want to address. I've listed these out, here, rather than including them in a formal review as it seems that not all of them result from changes in this PR:

- enable a myriad complementary studies -> enable myriad complementary studies
- and whose central coordinates places it inside -> and whose central coordinates place it inside 
- half these systems are too unresolved to be part -> half these systems are insufficiently resolved to be part ("too unresolved" sounded weird to me; but leave this unchanged if you prefer!)
- end up in the final SGA-2020 catalog (the Custom Mosaics & Ellipse-Fitting) -> end up in the final SGA-2020 catalog (see Custom Mosaics & Ellipse-Fitting)
- build a simple group catalog from the Final Parent Sample using a friends-of-friends ("Final Parent Sample," here, pretty much links to itself. I'd consider dropping the link).
- pixelized PSF in each bandpass. individual CCDs. (not sure why you need the trailing "individual CCDs" here?)
- measured half-light radius shape_r arcsec`; (there's a trailing backquote here)
- measured half-light radius shape_r arcsec`; (and here!)
- defined in the DR9 bitmasks page -> defined on the DR9 bitmasks page
- deviate by more than math:5-sigma (you're missing one of the backquote delimiters around the `math` clause).
- and the weight maps to variance images in (are the units correct here? I feel like the units of the weight maps should be 1/(the-images-squared).
- The principle SGA-2020 data product is the -> The principal SGA-2020 data product is the
- as described in the external catalogs documentation. (the link to "external catalogs documentation" appears to be broken? Although, maybe it will be present once Ben puts all of the pages in place?)

**Comment by moustakas** [comment]
A mirror of the documentation with this PR can be viewed here--
  https://portal.nersc.gov/project/cosmo/temp/ioannis/sga-dr9/sga/sga2020/

**Comment by moustakas** [comment]
@weaverba137 is this PR OK to merge, even if we don't immediately deploy it?

**Review by weaverba137 — APPROVED** [review]
This all looks fine.  Thank you for updating the change log.  Merge when ready.

**Pull request merged**